modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Actions.tsx

```diff
@@ -3,29 +3,338 @@
+import ClayTabs from '@clayui/tabs';
+import React, {useState} from 'react';
+
+import OrderableTable from '../components/OrderableTable';
+
+const types = [
+	{
+		label: 'Async',
+		value: 'async',
+	},
+	{
+		label: 'Headless',
+		value: 'headless',
+	},
+	{
+		label: 'Link',
+		value: 'link',
+	},
+	{
+		label: 'Modal',
+		value: 'modal',
+	},
+	{
+		label: 'Side Panel',
+		value: 'side-panel',
+	},
+];
+
+const messageTypes = [
+	{
+		label: 'Info',
+		value: 'info',
+	},
+	{
+		label: 'Secondary',
+		value: 'secondary',
+	},
+	{
+		label: 'Success',
+		value: 'success',
+	},
+	{
+		label: 'Danger',
+		value: 'danger',
+	},
+	{
+		label: 'Warning',
+		value: 'warning',
+	},
+];
+
+const noop = () => {};
 
 const Actions = () => {
+	const [activeBreadcrumb, setActiveBreadcrumb] = useState(0);
+	const [activeTab, setActiveTab] = useState(0);
+
 	return (
-		<ClayLayout.Sheet className="mt-3" size="lg">
-			<ClayLayout.SheetHeader>
-				<h2 className="sheet-title">
-					{Liferay.Language.get('actions')}
-				</h2>
-			</ClayLayout.SheetHeader>
-
-			<ClayLayout.SheetFooter>
-				<ClayButton.Group spaced>
-					<ClayButton>{Liferay.Language.get('save')}</ClayButton>
-
-					<ClayButton displayType="secondary">
-						{Liferay.Language.get('cancel')}
-					</ClayButton>
-				</ClayButton.Group>
-			</ClayLayout.SheetFooter>
-		</ClayLayout.Sheet>
+		<ClayLayout.ContainerFluid>
+			<ClayBreadcrumb
+				items={[
+					{
+						active: activeBreadcrumb === 0,
+						href: '#1',
+						label: Liferay.Language.get('actions'),
+					},
+					{
+						active: activeBreadcrumb === 1,
+						href: '#2',
+						label: Liferay.Language.get('new-item-action'),
+					},
+				]}
+			/>
+
+			<ClayLayout.ContainerFluid className="bg-white mb-4 p-0 rounded-sm">
+				{activeBreadcrumb === 0 && (
+					<>
+						<h2 className="mb-0 p-4">
+							{Liferay.Language.get('actions')}
+						</h2>
+
+						<ClayTabs
+							active={activeTab}
+							onActiveChange={setActiveTab}
+						>
+							<ClayTabs.Item>
+								{Liferay.Language.get('item-actions')}
+							</ClayTabs.Item>
+
+							<ClayTabs.Item>
+								{Liferay.Language.get('creation-actions')}
+							</ClayTabs.Item>
+
+							<ClayTabs.Item>
+								{Liferay.Language.get('bulk-actions')}
+							</ClayTabs.Item>
+						</ClayTabs>
+
+						<ClayTabs.Content active={activeTab} fade>
+							<ClayTabs.TabPane aria-labelledby="tab-1">
+								<OrderableTable
+									className="mt-0 p-1"
+									fields={[
+										{
+											label: Liferay.Language.get('icon'),
+											name: 'icon',
+										},
+										{
+											label: Liferay.Language.get(
+												'label'
+											),
+											name: 'label',
+										},
+										{
+											label: Liferay.Language.get('type'),
+											name: 'type',
+										},
+									]}
+									items={[]}
+									noItemsButtonLabel={Liferay.Language.get(
+										'create-item-action'
+									)}
```

Comment:
Not required in this task, but we should apply to the `OrdereableTable` the same `emptyState` API that we recently added to FDS.

Reply:
I opened a task for this: https://liferay.atlassian.net/browse/LPS-194347